src/openvdb.imageio/openvdbinput.cpp

```diff
@@ -332,22 +332,27 @@ openVDB(const std::string& filename, const ImageInput* errReport)
     if (magic != OPENVDB_MAGIC)
         return nullptr;
 
-    static struct OpenVDBLib {
-        OpenVDBLib() { openvdb::initialize(); }
-        ~OpenVDBLib() { openvdb::uninitialize(); }
-    } sVDBLib;
-
-    VDBFile file(new io::File(filename));
+    const char* errhint = "Unknown error";
     try {
+
+        static struct OpenVDBLib {
+            OpenVDBLib() { openvdb::initialize(); }
+            ~OpenVDBLib() { openvdb::uninitialize(); }
+        } sVDBLib;
+
+        VDBFile file(new io::File(filename));
+
         file->open();
         if (file->isOpen())
             return file;
+
     } catch (const std::exception& e) {
         errReport->error("Could not open '%s': %s", filename, e.what());
```

Comment:
If this is caught here, will you end up with two error messages? One here, then a second below?